### DISCUSSION TITLE
QuickAdd use with Task Plugin

### DISCUSSION THREAD

**Discussion by moorebi** [chhoumann/quickadd#91] [General]
I have been tinkering around with trying to setup QuickAdd to launch the Task Plugin (Obsidian Command) and enter the task typed with due date etc... from the "dialog" box that comes up when you run the `Tasks: Create or Edit Task` (below)... the trick here is to put the data entered when running the `Tasks: Create or Edit Task` command into the Daily Note file that is formatted at `YYYY-MM-DD.md` after the line `## Tasks`

![image](https://user-images.githubusercontent.com/6247939/131035640-b36fb613-001e-4969-8bda-c70a93a75863.png)

![image](https://user-images.githubusercontent.com/6247939/131035767-30e064a5-16c9-40a8-8e99-b793bf75193d.png)

Any thoughts comments suggestions.

**Comment by chhoumann** [comment]
Hi @moorebi!

I'm creating tasks (for the Tasks plugin) with this capture:
![image](https://user-images.githubusercontent.com/29108628/131290317-22343dae-0c94-44ed-975e-41e6808a51f3.png)
![image](https://user-images.githubusercontent.com/29108628/131290325-03f4e8b0-2de0-42f6-a4a5-5a5f2677543d.png)

Here's the Capture format, so you can copy-paste:
``{{VALUE}} 📆 {{DATE:YYYY-MM-DD}}``

My capture sends it to my tasks note with the deadline set to today. You can also use the following Capture format to make it ask you for a date, where you can just write `tomorrow`, and it'll register it as such:
``{{VALUE}} 📆 {{VDATE:📅 Date,YYYY-MM-DD}}``
But do keep in mind that feature requires the Natural Language Dates plugin.


And in your case, you'd need to use the Insert after feature, as well.
To send it to your current daily note, you can use the ``{{DATE}}`` format syntax. This is mine, as an example:
``bins/daily/{{DATE:gggg-MM-DD - ddd MMM D}}.md``

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. The capture recipe above still works verbatim on current QuickAdd, and two things have gotten better since: `{{VDATE}}` no longer needs the Natural Language Dates plugin (date parsing is built in now, so "tomorrow" or short aliases like `tm` just work), and Insert After now has a "Create line if not found" option, so `## Tasks` gets created automatically in daily notes that don't have it yet. Details: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice - closing as resolved.
